Homepage and introduction improvements (#432)
census-instrumentation/opencensus-website: content/introduction/_index.md

#### Time-series Metrics Collection
OpenCensus captures critical time series statistics from your application, including the latency, request count, and request size for each endpoint. Once captured, these individual statistics can be aggregated into metrics with time windows and dimensions of your choosing.

#### APIs
OpenCensus provides APIs for all telemetry types. For example, you can use these to define and capture custom metrics, add additional spans or annotations to traces, define custom trace sampling policy, switch context propagation formats, etc.

#### Integrations
OpenCensus contains an incredible amount of integrations out of the box. These allow traces and metrics to be captured from popular RPC systems, web frameworks, and storage clients.

#### Single Project and Implementation

> While OpenCensus is an open source project with an incredibly active community, it also benefits from maintaining a single distribution for each language. This means that contributions are focused, that efforts are wasted on multiple competing implementations (particularly important for integrations), and that as much functionality is available out of the box as possible.

Reviewer: s/that efforts are wasted on multiple/that efforts are not wasted on multiple/g

Author: ```suggestion
While OpenCensus is an open source project with an incredibly active community, it also benefits from maintaining a single distribution for each language. This means that contributions are focused, that efforts are not wasted on multiple competing implementations (particularly important for integrations), and that as much functionality is available out of the box as possible.
```

Author: Wow, thanks! Fixed

weight: 1
---

> {{<title>}} makes getting critical telemetry out of your services easy and automatic. Its origins stem from the rewrite of what has been Google's observability systems for the past 10 years. OpenCensus currently provides libraries for a number of languages that allow you to capture, manipulate, and export metrics and distributed traces to the backend(s) of your choice.

Reviewer: Please keep the mention of "vendor-agnosticity", being "an open source community developed project" plus "for microservices and monoliths alike". I've directly interacted with many people who just think that this is an entirely Google project but also that they don't care about microservices.

Author: ```suggestion
{{<title>}} makes getting critical telemetry out of your services easy and automatic. OpenCensus currently provides libraries for a number of languages that allow you to capture, manipulate, and export metrics and distributed traces to the backend(s) of your choice. OpenCensus works great with all software systems, from client applications, large monoliths, or highly-distributed microservices. OpenCensus started at Google but is now developed by a broad community of service developers, cloud vendors, and community contributors.
```

Reviewer: Nice, SGTM!

In this section we will walk through what OpenCensus is, what problems it solves, and how it can help your project.

{{% children %}}

Or, if you are ready to integrate OpenCensus in to your project, visit the [Quickstart](/quickstart).

> 

Reviewer: This is critical for community and brand association, why this removal?

Author: I'd originally removed it as it was already on the homepage, but after reviewing I agree that we should leave it

Author: ... as someone who's a bit of an idiot when it comes to Git, what's the best way to revert this change?

Reviewer: You can just recopy back from the original source https://raw.githubusercontent.com/census-instrumentation/opencensus-website/master/content/introduction/_index.md
otherwise you'd have to go through your git history

Author: Re-added

#### APIs
OpenCensus provides APIs for all telemetry types. For example, you can use these to define and capture custom metrics, add additional spans or annotations to traces, define custom trace sampling policy, switch context propagation formats, etc.

#### Integrations
OpenCensus contains an incredible amount of integrations out of the box. These allow traces and metrics to be captured automatically from popular RPC systems, web frameworks, and storage clients.

#### Single Project and Implementation
While OpenCensus is an open source project with an incredibly active community, it also benefits from maintaining a single distribution for each language. This means that contributions are focused, that efforts are wasted on multiple competing implementations (particularly important for integrations), and that as much functionality is available out of the box as possible.

#### Low Overhead
OpenCensus is used in production at some of the largest companies in the world, and as such it strives to have as small of a performance impact as possible.

#### Backend Support

> OpenCensus can upload data to almost any backend with its various exporter implementations. If you're using a backend that isn't supported yet (such as an analysis system built specifically for your organization), you can also implement your own quickly and easily. [Read more](/core-concepts/exporters/).

Reviewer: Invalid URL, please see `/exporters`

Author: Fixed:

```suggestion
OpenCensus can upload data to almost any backend with its various exporter implementations. If you're using a backend that isn't supported yet (such as an analysis system built specifically for your organization), you can also implement your own quickly and easily. [Read more](/exporters).
```
